10-std.tex

```diff
@@ -442,14 +442,14 @@ \section{Template}
 The console will trace \ic{The users are Mark(30) John(45)}.
 
 \paragraph{Macros}
-To callback code while parts of the template are rendered, providing a \expr{macros} object to the argument of \expr{Template.execute}. The key will act the template variable name, the value refers to a callback function that should return a \type{String}. The first argument of this macro function is always a \expr{resolve()} method, followed by the given arguments. The resolve function can be called to retreive values from the Template context. If \expr{macros} has no such field, the result is unspecified.
+To callback code while parts of the template are being rendered, provide a \expr{macros} object to the argument of \expr{Template.execute}. The key will act as the template variable name, the value refers to a callback function that should return a \type{String}. The first argument of this macro function is always a \expr{resolve()} method, followed by the given arguments. The resolve function can be called to retrieve values from the template context. If \expr{macros} has no such field, the result is unspecified.
 
-The following example passes itself as macro function context, and executes \ic{display} from the template.
+The following example passes itself as macro function context and executes \ic{display} from the template.
 \haxe{assets/TemplateMacros.hx}
 The console will trace \ic{The results: Mark ran 3.5 kilometers in 15 minutes}.
 
 \paragraph{Globals}
-Use the \expr{Template.globals} object to store those values that should be applied across all Template instances. This has lower priority than the context argument of \expr{Template.execute}.
+Use the \expr{Template.globals} object to store the values that should be applied across all template instances. This has lower priority than the context argument of \expr{Template.execute}.
```

Comment:
Also I would lose the "the" that used to be "those" entirely. "object to store values that"